frontend/src/app/problems/create/[problemId]/step/[step]/page.tsx

```
@@ -121,8 +122,16 @@ const Page = async ({params}: { params: Promise<{ problemId: string, step: strin
                                              executionConstraints={problemData.data.executionConstraints}
                 />
             case 3:
-                return <ReferenceSolution problemId={problemId}
-                                          languages={problemData.data.languages}/>
+                const presignedUrlResponse = await getPresignedUrl(problemId,
+                    headersList.get('Authorization') || "");
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)